src/styles/utils/variables.scss

```diff
@@ -0,0 +1,28 @@
+$clock-height: 80vmin;
```

Comment:
why do you need two variables with the same value?

Reply:
Thanks for the answer. I fixed this error. But I was guided by the video "When to use variables" in the topic about sass (1 min, 20 sec)